wgpu/gui/offscreen.py

```diff
@@ -62,3 +64,18 @@ def draw(self):
 
 
 WgpuCanvas = WgpuManualOffscreenCanvas
+
+
+def call_later(delay, callback, *args):
+    loop = asyncio.get_event_loop()
```

Comment:
They seem to want to get rid of `get_event_loop` in favor of `get_running_loop` and `run`.
```suggestion
    if hasattr(asyncio, "get_running_loop"):
        loop = asyncio.get_running_loop()
    else:  # py3.6 or lower
        loop = asyncio.get_event_loop()
```

Reply:
Actually the equivalent seems to be `asyncio.get_event_loop_policy().get_event_loop()`